packages/service-library/src/servicelib/utils.py

```diff
@@ -113,8 +112,10 @@ async def sem_task(task: Awaitable[Any]) -> Any:
                 return await task
 
         wrapped_tasks = [sem_task(t) for t in tasks]
+    else:
+        wrapped_tasks = tasks
```

Comment:
what is the advantage in moving this here? I would have done it as it was before.

Reply:
linter suggested that